### PR TITLE
Improve package version changing

### LINKED ISSUE #620: Implement change to speed up upgrade/alt processing
Switching releases (whether upgrade or alternating) can take a considerable time for packages with a lot of files.
Raising issue to push change to speed up this processing..

### LINKED ISSUE #623: a prompt for zopen remove
When removing a package, there could/should be a prompt similar to "install" along with a -y.

### PR DESCRIPTION
**Pull request by DevonianTeuchter**
Should improve upgrade times [zopen upgrade] and switching to alternative versions [zopen alt...] .
Adds in question about removing packages (with y/n/a answers - a meaning to answer yes to all upcoming questions 
Removes "parallel" checking logic as obsolete and fragile.

resolves #623 
resolves #620

### CODE REVIEW

File: include/common.sh [hunk 1]
```diff
@@ -571,113 +602,99 @@ unsymlinkFromSystem()
...
-      alreadyfound=""
-    fi
-  elif [ -L "${filename}" ]; then
-    if [ ! -f "${filename}" ]; then
-      # the linked-to file no longer exists (ie. the symlink is dangling)
-      rm -f "${filename}" > /dev/null 2>&1
     fi
   else
-    echo "Unprocessable file: '${filename}'" >> "${tempTrash}"
+    printDebug "No list of current links to cgeck - package was not installed/active"
```
Comment (IgorTodorovskiIBM) [inline include/common.sh:694]:
```suggestion
    printDebug "No list of current links to check - package was not installed/active"
```

**Review by IgorTodorovskiIBM — APPROVED** [review]
Nice improvement! LGTM

**Comment by DevonianTeuchter** [comment]
Awaiting ZOSOpenTools/metaport#47 to run tests

**Pull request merged**